android/app/src/main/java/app/covidshield/module/ExposureNotificationModule.kt

```diff
@@ -96,6 +103,10 @@ class ExposureNotificationModule(context: ReactApplicationContext) : ReactContex
     @ReactMethod
     fun detectExposure(configuration: ReadableMap, diagnosisKeysURLs: ReadableArray, promise: Promise) {
         promise.launch(this) {
+            if (getStatusInternal() != Status.ACTIVE) {
```

Comment:
I think the use of `getStatusInternal()` at this point is overreaching what is required. It could be possible to perform  the `detectExposure` functions with **Bluetooth** turned off, but with the **exposureNotificationClient** enabled.

Reply:
You are right. I just tested on Android when EN is ON but Bluetooth is OFF, the framework still can run `detectExposure`. However, I wouldn't think it is important to separate the condition between EN and Bluetooth. It's clearly that in order for the app to work properly, both EN and Bluetooth have to be ON. I prefer to keep same logic for consistency.

Reply:
@jeberhardt points out this

> Some people may turn off Bluetooth to "save battery", like when they are at home, and only turn it on when they go out. It's possible. I don't think consistency in the logic should be more important than specific functionality.

I will test all changes again with just EN.

Reply:
As a result of this https://github.com/cds-snc/covid-alert-app/pull/866#pullrequestreview-454582620, I would prefer to check both EN + Bluetooth. This is a low risk approach, imo.

Reply:
Tested again with just checking EN. It works fine. 👍 cc @jeberhardt @timarney